web/client/components/app/StandardApp.jsx

```diff
@@ -47,6 +47,15 @@ class StandardApp extends React.Component {
         store: null
     };
 
+    addProjDefinitions(config) {
+        if (config.projectionDefs && config.projectionDefs.length) {
+            config.projectionDefs.forEach((proj) => {
+                window.proj4.defs(proj.code, proj.def);
```

Comment:
You should be able to require proj4 from web/client/libs as usual and use that one instead of window.proj4

Reply:
👍